### ISSUE TITLE
Show quick "help" on how to load the data

### ISSUE THREAD

**Issue by darribas** [urbangrammarai/signatures_gb#1]
@martinfleis it'd be nice to have on the `README` a short line of code showing how the dataset can be loaded when stored locally. I'm just thinking something along the lines of:

```python
import dask_geopandas

db = dask_geopandas.read_parquet('XXX')
```

What else would a user need to get up and running with accessing the data? If you sketch it out on this issue, I can dress it up and add it to the `README`.

**Comment by martinfleis** [comment]
1. clone the repo
2. navigate to the repo
```
cd signatures_gb
```
3. open python and read the files
```py
import dask_geopandas

db = dask_geopandas.read_parquet('form')
```

That should be all.

**Comment by darribas** [comment]
Implemented in [7af8d49](https://github.com/urbangrammarai/signatures_gb/commit/7af8d495bac51cf39ce37ee4d6bdae725d881c0e)
